web/templates/user/_user_show_traffic.html

```diff
@@ -1,4 +1,4 @@
-{#
```

Comment:
Why is there a change here? Did you add/remove a byte-order mark?

Reply:
@JuKu 
There is still a change there for no reason.